scripts/combine/setupCombineWMass.py

```diff
@@ -83,6 +85,43 @@ def main(args):
     if not os.path.isdir(outfolder):
         os.makedirs(outfolder)
 
+    unconstrainedProcs = []
```

Comment:
This should probably go in a separate function to avoid making this script to buys

Reply:
I moved most of the product into the defineSignalBinsUnfolding function defined in the datagroups class. The code should be generic and I think can be used also for the low PU

Reply:
yes agreed.